avocado/utils/multipath.py

```diff
@@ -106,6 +106,37 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
+    :return: Multipath wwid
+    :rtype: str
+    """
+    cmd = "multipathd show maps format '%n %w'"
+    wwids = process.run(cmd, ignore_status=True,
+                        sudo=True, shell=True).stdout_text
+    for wwid in wwids.splitlines(): 
+        if mpath in wwid:
+            return wwid.split()[1]
+
+
+def is_mpath(mpath):
+    """
+    Check the give name is a multipath device name or not.
+
+    :return: True if device is multipath or False
+    :rtype: Boolean
+    """
+    cmd = "multipath -l -v 1"
+    mpaths = process.run(cmd, ignore_status=True,
+                        sudo=True, shell=True).stdout_text
```

Comment:
I think that there is a trailing space here and/or an indentation problem that is making Travis complain.

Reply:
fixed the indentation issue.